public/app/plugins/datasource/opentsdb/datasource.js

```diff
@@ -162,12 +162,21 @@ function (angular, _, dateMath) {
       });
     };
 
-    this._performMetricKeyValueLookup = function(metric, key) {
-      if(!metric || !key) {
+    this._performMetricKeyValueLookup = function(metric, keys) {
+
+      if(!metric || !keys) {
         return $q.when([]);
       }
 
-      var m = metric + "{" + key + "=*}";
+      var keysArray = keys.split(",");
+      var key = keysArray[0];
+      var keysQuery = key + "=*";
+
+      if (keysArray.length > 1) {
+        keysQuery += "," + keysArray.splice(1).join(",");
```

Comment:
what about trimming spaces? OpenTSDB doesn't allow leading or trailing spaces in this part of query